### DISCUSSION TITLE
Planned 1.9.0 release date?

### DISCUSSION THREAD

**Discussion by torokati44** [wjakob/nanobind#411] [General]
With the release of 1.8.0 having been a while ago, and some fixes already in `master` since then (some quite recently even); I'm just wondering whether we should update to 1.8.0 now, or wait for 1.9.0.
Do you have any plans or guesses on when that will come out?

**Comment by wjakob** [comment]
I want to get in https://github.com/wjakob/nanobind/pull/335 before making a new release. And I did not yet have time to do it.

**Reply by torokati44** [reply]
I see. Thank you!

**Reply by wjakob** [reply]
FYI, I just released version 1.9.0.

**Reply by torokati44** [reply]
Awesome!
